hawkbit-runtime/hawkbit-update-server/src/test/java/org/eclipse/hawkbit/app/AllowedHostNamesTest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.app;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.security.web.firewall.RequestRejectedException;
+
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource(properties = { "hawkbit.server.security.allowedHostNames=localhost",
+        "hawkbit.server.security.httpFirewallIgnoredPaths=/index.html" })
+@Feature("Integration Test - Security")
+@Story("Allowed Host Names")
+public class AllowedHostNamesTest extends AbstractSecurityTest {
+
+    @Test
+    public void allowedHostNameWithNotAllowedHost() {
```

Comment:
Please add a proper Allure description to the tests

Reply:
Done.